fedn/fedn/common/control/package.py

```diff
@@ -212,19 +212,14 @@ def dispatcher(self, run_path):
         :param run_path:
         :return:
         """
-        dispatch_dir = self.dir
         from_path = os.path.join(os.getcwd(), 'client')
-        import time
-        dirname = time.strftime("%Y%m%d-%H%M%S")
-
+        
         from distutils.dir_util import copy_tree
         copy_tree(from_path, run_path)
 
-        os.chdir(run_path)
```

Comment:
Why can we remove this?

Reply:
Is not the question, why is it there? Absolute paths are being used everwhere, even in client.py:
``` 
from fedn.common.control.package import PackageRuntime
pr = PackageRuntime(os.getcwd(), os.getcwd())
``` 

Why is it like this?

Reply:
@morganekmefjord Maybe you remember?

Reply:
This PR has been closed due to build failure, please continue review in PR #290